agents/node/vcxagent-core/src/agent.js

```diff
@@ -34,8 +35,9 @@ async function createVcxAgent ({ agentName, genesisPath, agencyUrl, seed, wallet
   const agentProvision = await storageService.loadAgentProvision()
   const issuerDid = agentProvision.issuerConfig.institution_did
 
-  async function agentInitVcx () {
+  async function agentInitVcx() {
```

Comment:
Just dropping here as note, we should create issue to replace usage of `serviceConnections`  by `serviceNonmediatedConnections` everywhere. Then we can also drop agency client init code, starting up agency in nodejs tests

Reply:
By "everywhere", I assume you mean the `vcxagent-core` integration tests (the current Alice and Faber agents)? Although, since I think that you would agree that mediation should remain optional for `vcxagent-core` clients for a while, we should perhaps also leave at least some coverage for the mediated connection?